app/validate_txs.go

```diff
@@ -54,6 +56,13 @@ func filterStdTxs(logger log.Logger, dec sdk.TxDecoder, ctx sdk.Context, handler
 		// Set the tx size on the context before calling the AnteHandler
 		ctx = ctx.WithTxBytes(tx)
 
+		msgTypes := msgTypes(sdkTx)
+		if sdkTransactionsCount+len(sdkTx.GetMsgs()) > appconsts.NonPFBTransactionCap {
+			logger.Debug("skipping tx because the sdk message cap was reached", "tx", tmbytes.HexBytes(coretypes.Tx(tx).Hash()))
+			continue
+		}
+		sdkTransactionsCount += len(sdkTx.GetMsgs())
```

Comment:
[optional] rename for consistency `sdkTransactionsCount` => `nonPFBTransactionCount`

Reply:
01e4cb4cf6c75a4567a9c88c609e32c0bcebd6c5